libs/ledger-live-common/src/families/algorand/walletApiAdapter.ts

```diff
@@ -0,0 +1,23 @@
+import { AlgorandTransaction as WalletAPITransaction } from "@ledgerhq/wallet-api-core";
+import {
+  AreFeesProvided,
+  GetWalletAPITransactionSignFlowInfos,
+} from "../../wallet-api/types";
+import { Transaction } from "./types";
+
+const CAN_EDIT_FEES = true;
+
+const areFeesProvided: AreFeesProvided<WalletAPITransaction> = (tx) =>
+  !!tx.fees;
+
+const getWalletAPITransactionSignFlowInfos: GetWalletAPITransactionSignFlowInfos<
+  WalletAPITransaction,
+  Transaction
+> = (tx) => ({
+  canEditFees: CAN_EDIT_FEES,
+  // TODO: Why AlgorandOperationMode is different (e.g. `optOut`)?
```

Comment:
There is a diff in `AlgorandOperationMode` between one live-common and wallet-api (e.g. `optOut`).